app/models/concerns/publishable.rb

```diff
@@ -14,27 +14,25 @@ module Publishable
 
   def publish!
     publish_content_item!
-    publish_rummager_artefact!
+    add_to_search_index!
+  end
+
+  def base_path
+    "/government/policies/#{slug}"
```

Comment:
Ah, I didn't realise you were using this concern for email signups as well. I think ultimately this concerns days are numbered. I think eventually we're going to end up with a single generic `Policy` model (that has a `base_path` method) and there will be a separate service object to perform the post-save actions of stuffing it in content store and indexing it in search.

I can move the `base_path` method onto the `PolicyArea` and `Programme` in the mean time if it will help with the email signup work.